Modules Release 4.8.0 (2021-07-14)
DCC-SW: Added modules (2023-aug/XeonE5-2660v3)
DCC-SW: Removed standard modules: /apps/dcc/etc/ModulesSL73/modulefiles
DCC-SW: Using gnu/12.3.0 compiler (requested)
DCC-SW: Released modules (2023-aug/XeonE5-2660v3)
DCC-SW: Added modules (2020-aug/XeonE5-2660v3)
DCC-SW: Removed standard modules: /apps/dcc/etc/ModulesSL73/modulefiles
DCC-SW: Using gnu/9.3.0 compiler
Loading python/3.8.5
  Loading requirement: build-tools/1.0 gcc/9.3.0 zlib/1.2.11 expat/2.2.0
    libffi/3.3
Loading scalapack/2.1.0
  Loading requirement: numactl/2.0.12 libxml2/2.9.10 hwloc/2.2.0 knem/1.1.4
    ucx/1.8.1 openmpi/4.0.4
mpiexec: Forwarding signal 18 to job
mpiexec: Forwarding signal 18 to job
Error in sys.excepthook:
Traceback (most recent call last):
  File "/zhome/87/8/147075/venv-name/gpaw/gpaw/mpi.py", line 1197, in print_mpi_stack_trace
    def print_mpi_stack_trace(type, value, tb):
KeyboardInterrupt

Original exception was:
Traceback (most recent call last):
  File "soc.py", line 62, in <module>
    en_soc = soc_eigenstates(calc=calc, projected=True, theta=theta, phi=phi,
  File "/zhome/87/8/147075/venv-name/gpaw/gpaw/spinorbit.py", line 500, in soc_eigenstates
    dVL_avii = {a: soc(calc.wfs.setups[a],
  File "/zhome/87/8/147075/venv-name/gpaw/gpaw/spinorbit.py", line 501, in <dictcomp>
    calc.hamiltonian.xc, D_sp) * scale
  File "/zhome/87/8/147075/venv-name/gpaw/gpaw/new/ase_interface.py", line 476, in hamiltonian
    return FakeHamiltonian(self.calculation)
  File "/zhome/87/8/147075/venv-name/gpaw/gpaw/new/backwards_compatibility.py", line 183, in __init__
    self.finegd = self.pot_calc.fine_grid._gd
  File "/zhome/87/8/147075/venv-name/gpaw/gpaw/core/uniform_grid.py", line 227, in _gd
    return GridDescriptor(self.size_c,
  File "/zhome/87/8/147075/venv-name/gpaw/gpaw/grid_descriptor.py", line 120, in __init__
    Domain.__init__(self, cell_cv, pbc_c, comm, parsize_c, self.N_c)
  File "/zhome/87/8/147075/venv-name/gpaw/gpaw/domain.py", line 47, in __init__
    self.xxxiucell_cv = np.linalg.inv(self.xxxucell_cv.T)  # Jacobian
  File "<__array_function__ internals>", line 200, in inv
  File "/zhome/87/8/147075/venv-name/lib/python3.8/site-packages/numpy/linalg/linalg.py", line 538, in inv
    ainv = _umath_linalg.inv(a, signature=signature, extobj=extobj)
KeyboardInterrupt
GPAW CLEANUP (node 0): <class 'KeyboardInterrupt'> occurred.  Calling MPI_Abort!
Error in sys.excepthook:
Traceback (most recent call last):
  File "/zhome/87/8/147075/venv-name/gpaw/gpaw/mpi.py", line 1197, in print_mpi_stack_trace
    def print_mpi_stack_trace(type, value, tb):
KeyboardInterrupt

Original exception was:
Traceback (most recent call last):
  File "soc.py", line 62, in <module>
    en_soc = soc_eigenstates(calc=calc, projected=True, theta=theta, phi=phi,
  File "/zhome/87/8/147075/venv-name/gpaw/gpaw/spinorbit.py", line 500, in soc_eigenstates
    dVL_avii = {a: soc(calc.wfs.setups[a],
  File "/zhome/87/8/147075/venv-name/gpaw/gpaw/spinorbit.py", line 500, in <dictcomp>
    dVL_avii = {a: soc(calc.wfs.setups[a],
  File "/zhome/87/8/147075/venv-name/gpaw/gpaw/spinorbit.py", line 545, in soc
    v_g = get_radial_potential(a, xc, D_sp)
  File "/zhome/87/8/147075/venv-name/gpaw/gpaw/spinorbit.py", line 599, in get_radial_potential
    n_sg = np.dot(D_sq, n_qg) / (4 * np.pi)**0.5
  File "<__array_function__ internals>", line 200, in dot
KeyboardInterrupt
GPAW CLEANUP (node 1): <class 'KeyboardInterrupt'> occurred.  Calling MPI_Abort!
Error in sys.excepthook:
Traceback (most recent call last):
  File "/zhome/87/8/147075/venv-name/gpaw/gpaw/mpi.py", line 1197, in print_mpi_stack_trace
    def print_mpi_stack_trace(type, value, tb):
KeyboardInterrupt

Original exception was:
Traceback (most recent call last):
  File "soc.py", line 62, in <module>
    en_soc = soc_eigenstates(calc=calc, projected=True, theta=theta, phi=phi,
  File "/zhome/87/8/147075/venv-name/gpaw/gpaw/spinorbit.py", line 500, in soc_eigenstates
    dVL_avii = {a: soc(calc.wfs.setups[a],
  File "/zhome/87/8/147075/venv-name/gpaw/gpaw/spinorbit.py", line 501, in <dictcomp>
    calc.hamiltonian.xc, D_sp) * scale
  File "/zhome/87/8/147075/venv-name/gpaw/gpaw/new/ase_interface.py", line 476, in hamiltonian
    return FakeHamiltonian(self.calculation)
  File "/zhome/87/8/147075/venv-name/gpaw/gpaw/new/backwards_compatibility.py", line 183, in __init__
    self.finegd = self.pot_calc.fine_grid._gd
  File "/zhome/87/8/147075/venv-name/gpaw/gpaw/core/uniform_grid.py", line 227, in _gd
    return GridDescriptor(self.size_c,
  File "/zhome/87/8/147075/venv-name/gpaw/gpaw/grid_descriptor.py", line 157, in __init__
    np.diag(self.cell_cv.diagonal())).any()
  File "<__array_function__ internals>", line 200, in diag
  File "/zhome/87/8/147075/venv-name/lib/python3.8/site-packages/numpy/lib/twodim_base.py", line 298, in diag
    n = s[0]+abs(k)
KeyboardInterrupt
GPAW CLEANUP (node 2): <class 'KeyboardInterrupt'> occurred.  Calling MPI_Abort!
Error in sys.excepthook:
Traceback (most recent call last):
  File "/zhome/87/8/147075/venv-name/gpaw/gpaw/mpi.py", line 1197, in print_mpi_stack_trace
    def print_mpi_stack_trace(type, value, tb):
KeyboardInterrupt

Original exception was:
Traceback (most recent call last):
  File "soc.py", line 62, in <module>
    en_soc = soc_eigenstates(calc=calc, projected=True, theta=theta, phi=phi,
  File "/zhome/87/8/147075/venv-name/gpaw/gpaw/spinorbit.py", line 500, in soc_eigenstates
    dVL_avii = {a: soc(calc.wfs.setups[a],
  File "/zhome/87/8/147075/venv-name/gpaw/gpaw/spinorbit.py", line 500, in <dictcomp>
    dVL_avii = {a: soc(calc.wfs.setups[a],
  File "/zhome/87/8/147075/venv-name/gpaw/gpaw/spinorbit.py", line 560, in soc
    dVL_vii[0, N1:N1 + Nm, N2:N2 + Nm] = c * Lx_lmm[l1]
KeyboardInterrupt
GPAW CLEANUP (node 3): <class 'KeyboardInterrupt'> occurred.  Calling MPI_Abort!
Error in sys.excepthook:
Traceback (most recent call last):
  File "/zhome/87/8/147075/venv-name/gpaw/gpaw/mpi.py", line 1197, in print_mpi_stack_trace
    def print_mpi_stack_trace(type, value, tb):
KeyboardInterrupt

Original exception was:
Traceback (most recent call last):
  File "soc.py", line 62, in <module>
    en_soc = soc_eigenstates(calc=calc, projected=True, theta=theta, phi=phi,
  File "/zhome/87/8/147075/venv-name/gpaw/gpaw/spinorbit.py", line 500, in soc_eigenstates
    dVL_avii = {a: soc(calc.wfs.setups[a],
  File "/zhome/87/8/147075/venv-name/gpaw/gpaw/spinorbit.py", line 500, in <dictcomp>
    dVL_avii = {a: soc(calc.wfs.setups[a],
  File "/zhome/87/8/147075/venv-name/gpaw/gpaw/spinorbit.py", line 545, in soc
    v_g = get_radial_potential(a, xc, D_sp)
  File "/zhome/87/8/147075/venv-name/gpaw/gpaw/spinorbit.py", line 617, in get_radial_potential
    xc.calculate_spherical(rgd, n_sg, v_sg)
  File "/zhome/87/8/147075/venv-name/gpaw/gpaw/xc/lda.py", line 111, in calculate_spherical
    e_g[:], dedn_sg = rcalc(rgd, n_sg[:, np.newaxis], [1.0])
  File "/zhome/87/8/147075/venv-name/gpaw/gpaw/xc/lda.py", line 81, in __call__
    self.kernel.calculate(e_g, n_sg, dedn_sg)
  File "/zhome/87/8/147075/venv-name/gpaw/gpaw/xc/noncollinear.py", line 26, in calculate
    vnew_sg /= np.where(m_g < 1e-15, 1, m_g)
  File "<__array_function__ internals>", line 179, in where
KeyboardInterrupt
GPAW CLEANUP (node 4): <class 'KeyboardInterrupt'> occurred.  Calling MPI_Abort!
Error in sys.excepthook:
Traceback (most recent call last):
  File "/zhome/87/8/147075/venv-name/gpaw/gpaw/mpi.py", line 1197, in print_mpi_stack_trace
    def print_mpi_stack_trace(type, value, tb):
KeyboardInterrupt

Original exception was:
Traceback (most recent call last):
  File "soc.py", line 62, in <module>
    en_soc = soc_eigenstates(calc=calc, projected=True, theta=theta, phi=phi,
  File "/zhome/87/8/147075/venv-name/gpaw/gpaw/spinorbit.py", line 500, in soc_eigenstates
    dVL_avii = {a: soc(calc.wfs.setups[a],
  File "/zhome/87/8/147075/venv-name/gpaw/gpaw/spinorbit.py", line 500, in <dictcomp>
    dVL_avii = {a: soc(calc.wfs.setups[a],
  File "/zhome/87/8/147075/venv-name/gpaw/gpaw/spinorbit.py", line 545, in soc
    v_g = get_radial_potential(a, xc, D_sp)
  File "/zhome/87/8/147075/venv-name/gpaw/gpaw/spinorbit.py", line 610, in get_radial_potential
    fh_g = -np.array([np.sum(rho_g[:ig]) for ig in range(len(r_g))]) / r_g**2
  File "/zhome/87/8/147075/venv-name/gpaw/gpaw/spinorbit.py", line 610, in <listcomp>
    fh_g = -np.array([np.sum(rho_g[:ig]) for ig in range(len(r_g))]) / r_g**2
  File "<__array_function__ internals>", line 200, in sum
  File "/zhome/87/8/147075/venv-name/lib/python3.8/site-packages/numpy/core/fromnumeric.py", line 2324, in sum
    return _wrapreduction(a, np.add, 'sum', axis, dtype, out, keepdims=keepdims,
KeyboardInterrupt
GPAW CLEANUP (node 5): <class 'KeyboardInterrupt'> occurred.  Calling MPI_Abort!
Error in sys.excepthook:
Traceback (most recent call last):
  File "/zhome/87/8/147075/venv-name/gpaw/gpaw/mpi.py", line 1197, in print_mpi_stack_trace
    def print_mpi_stack_trace(type, value, tb):
KeyboardInterrupt

Original exception was:
Traceback (most recent call last):
  File "soc.py", line 62, in <module>
    en_soc = soc_eigenstates(calc=calc, projected=True, theta=theta, phi=phi,
  File "/zhome/87/8/147075/venv-name/gpaw/gpaw/spinorbit.py", line 500, in soc_eigenstates
    dVL_avii = {a: soc(calc.wfs.setups[a],
  File "/zhome/87/8/147075/venv-name/gpaw/gpaw/spinorbit.py", line 500, in <dictcomp>
    dVL_avii = {a: soc(calc.wfs.setups[a],
  File "/zhome/87/8/147075/venv-name/gpaw/gpaw/spinorbit.py", line 545, in soc
    v_g = get_radial_potential(a, xc, D_sp)
  File "/zhome/87/8/147075/venv-name/gpaw/gpaw/spinorbit.py", line 610, in get_radial_potential
    fh_g = -np.array([np.sum(rho_g[:ig]) for ig in range(len(r_g))]) / r_g**2
  File "/zhome/87/8/147075/venv-name/gpaw/gpaw/spinorbit.py", line 610, in <listcomp>
    fh_g = -np.array([np.sum(rho_g[:ig]) for ig in range(len(r_g))]) / r_g**2
  File "<__array_function__ internals>", line 200, in sum
  File "/zhome/87/8/147075/venv-name/lib/python3.8/site-packages/numpy/core/fromnumeric.py", line 2324, in sum
    return _wrapreduction(a, np.add, 'sum', axis, dtype, out, keepdims=keepdims,
  File "/zhome/87/8/147075/venv-name/lib/python3.8/site-packages/numpy/core/fromnumeric.py", line 70, in _wrapreduction
    passkwargs = {k: v for k, v in kwargs.items()
KeyboardInterrupt
GPAW CLEANUP (node 6): <class 'KeyboardInterrupt'> occurred.  Calling MPI_Abort!
rank=11 L00: Traceback (most recent call last):
rank=11 L01:   File "soc.py", line 62, in <module>
rank=11 L02:     en_soc = soc_eigenstates(calc=calc, projected=True, theta=theta, phi=phi,
rank=11 L03:   File "/zhome/87/8/147075/venv-name/gpaw/gpaw/spinorbit.py", line 500, in soc_eigenstates
rank=11 L04:     dVL_avii = {a: soc(calc.wfs.setups[a],
rank=11 L05:   File "/zhome/87/8/147075/venv-name/gpaw/gpaw/spinorbit.py", line 501, in <dictcomp>
rank=11 L06:     calc.hamiltonian.xc, D_sp) * scale
rank=11 L07:   File "/zhome/87/8/147075/venv-name/gpaw/gpaw/new/ase_interface.py", line 476, in hamiltonian
rank=11 L08:     return FakeHamiltonian(self.calculation)
rank=11 L09:   File "/zhome/87/8/147075/venv-name/gpaw/gpaw/new/backwards_compatibility.py", line 183, in __init__
rank=11 L10:     self.finegd = self.pot_calc.fine_grid._gd
rank=11 L11:   File "/zhome/87/8/147075/venv-name/gpaw/gpaw/core/uniform_grid.py", line 227, in _gd
rank=11 L12:     return GridDescriptor(self.size_c,
rank=11 L13:   File "/zhome/87/8/147075/venv-name/gpaw/gpaw/grid_descriptor.py", line 120, in __init__
rank=11 L14:     Domain.__init__(self, cell_cv, pbc_c, comm, parsize_c, self.N_c)
rank=11 L15:   File "/zhome/87/8/147075/venv-name/gpaw/gpaw/domain.py", line 55, in __init__
rank=11 L16:     self.set_decomposition(comm, parsize_c, N_c)
rank=11 L17:   File "/zhome/87/8/147075/venv-name/gpaw/gpaw/domain.py", line 77, in set_decomposition
rank=11 L18:     if np.prod(self.parsize_c) != self.comm.size:
rank=11 L19:   File "<__array_function__ internals>", line 200, in prod
rank=11 L20:   File "/zhome/87/8/147075/venv-name/lib/python3.8/site-packages/numpy/core/fromnumeric.py", line 3076, in prod
rank=11 L21:     return _wrapreduction(a, np.multiply, 'prod', axis, dtype, out,
rank=11 L22:   File "/zhome/87/8/147075/venv-name/lib/python3.8/site-packages/numpy/core/fromnumeric.py", line 86, in _wrapreduction
rank=11 L23:     return ufunc.reduce(obj, axis, dtype, out, **passkwargs)
rank=11 L24: KeyboardInterrupt
GPAW CLEANUP (node 11): <class 'KeyboardInterrupt'> occurred.  Calling MPI_Abort!
Error in sys.excepthook:
Traceback (most recent call last):
  File "/zhome/87/8/147075/venv-name/gpaw/gpaw/mpi.py", line 1197, in print_mpi_stack_trace
    def print_mpi_stack_trace(type, value, tb):
KeyboardInterrupt

Original exception was:
Traceback (most recent call last):
  File "soc.py", line 62, in <module>
    en_soc = soc_eigenstates(calc=calc, projected=True, theta=theta, phi=phi,
  File "/zhome/87/8/147075/venv-name/gpaw/gpaw/spinorbit.py", line 500, in soc_eigenstates
    dVL_avii = {a: soc(calc.wfs.setups[a],
  File "/zhome/87/8/147075/venv-name/gpaw/gpaw/spinorbit.py", line 500, in <dictcomp>
    dVL_avii = {a: soc(calc.wfs.setups[a],
  File "/zhome/87/8/147075/venv-name/gpaw/gpaw/spinorbit.py", line 545, in soc
    v_g = get_radial_potential(a, xc, D_sp)
  File "/zhome/87/8/147075/venv-name/gpaw/gpaw/spinorbit.py", line 610, in get_radial_potential
    fh_g = -np.array([np.sum(rho_g[:ig]) for ig in range(len(r_g))]) / r_g**2
  File "/zhome/87/8/147075/venv-name/gpaw/gpaw/spinorbit.py", line 610, in <listcomp>
    fh_g = -np.array([np.sum(rho_g[:ig]) for ig in range(len(r_g))]) / r_g**2
  File "<__array_function__ internals>", line 200, in sum
  File "/zhome/87/8/147075/venv-name/lib/python3.8/site-packages/numpy/core/fromnumeric.py", line 2324, in sum
    return _wrapreduction(a, np.add, 'sum', axis, dtype, out, keepdims=keepdims,
KeyboardInterrupt
GPAW CLEANUP (node 7): <class 'KeyboardInterrupt'> occurred.  Calling MPI_Abort!
Error in sys.excepthook:
Traceback (most recent call last):
  File "/zhome/87/8/147075/venv-name/gpaw/gpaw/mpi.py", line 1197, in print_mpi_stack_trace
    def print_mpi_stack_trace(type, value, tb):
KeyboardInterrupt

Original exception was:
Traceback (most recent call last):
  File "soc.py", line 62, in <module>
    en_soc = soc_eigenstates(calc=calc, projected=True, theta=theta, phi=phi,
  File "/zhome/87/8/147075/venv-name/gpaw/gpaw/spinorbit.py", line 500, in soc_eigenstates
    dVL_avii = {a: soc(calc.wfs.setups[a],
  File "/zhome/87/8/147075/venv-name/gpaw/gpaw/spinorbit.py", line 500, in <dictcomp>
    dVL_avii = {a: soc(calc.wfs.setups[a],
  File "/zhome/87/8/147075/venv-name/gpaw/gpaw/spinorbit.py", line 545, in soc
    v_g = get_radial_potential(a, xc, D_sp)
  File "/zhome/87/8/147075/venv-name/gpaw/gpaw/spinorbit.py", line 610, in get_radial_potential
    fh_g = -np.array([np.sum(rho_g[:ig]) for ig in range(len(r_g))]) / r_g**2
  File "/zhome/87/8/147075/venv-name/gpaw/gpaw/spinorbit.py", line 610, in <listcomp>
    fh_g = -np.array([np.sum(rho_g[:ig]) for ig in range(len(r_g))]) / r_g**2
  File "<__array_function__ internals>", line 200, in sum
  File "/zhome/87/8/147075/venv-name/lib/python3.8/site-packages/numpy/core/fromnumeric.py", line 2324, in sum
    return _wrapreduction(a, np.add, 'sum', axis, dtype, out, keepdims=keepdims,
  File "/zhome/87/8/147075/venv-name/lib/python3.8/site-packages/numpy/core/fromnumeric.py", line 86, in _wrapreduction
    return ufunc.reduce(obj, axis, dtype, out, **passkwargs)
KeyboardInterrupt
GPAW CLEANUP (node 8): <class 'KeyboardInterrupt'> occurred.  Calling MPI_Abort!
Error in sys.excepthook:
Traceback (most recent call last):
  File "/zhome/87/8/147075/venv-name/gpaw/gpaw/mpi.py", line 1197, in print_mpi_stack_trace
    def print_mpi_stack_trace(type, value, tb):
KeyboardInterrupt

Original exception was:
Traceback (most recent call last):
  File "soc.py", line 62, in <module>
    en_soc = soc_eigenstates(calc=calc, projected=True, theta=theta, phi=phi,
  File "/zhome/87/8/147075/venv-name/gpaw/gpaw/spinorbit.py", line 500, in soc_eigenstates
    dVL_avii = {a: soc(calc.wfs.setups[a],
  File "/zhome/87/8/147075/venv-name/gpaw/gpaw/spinorbit.py", line 500, in <dictcomp>
    dVL_avii = {a: soc(calc.wfs.setups[a],
  File "/zhome/87/8/147075/venv-name/gpaw/gpaw/spinorbit.py", line 545, in soc
    v_g = get_radial_potential(a, xc, D_sp)
  File "/zhome/87/8/147075/venv-name/gpaw/gpaw/spinorbit.py", line 610, in get_radial_potential
    fh_g = -np.array([np.sum(rho_g[:ig]) for ig in range(len(r_g))]) / r_g**2
  File "/zhome/87/8/147075/venv-name/gpaw/gpaw/spinorbit.py", line 610, in <listcomp>
    fh_g = -np.array([np.sum(rho_g[:ig]) for ig in range(len(r_g))]) / r_g**2
  File "<__array_function__ internals>", line 200, in sum
  File "/zhome/87/8/147075/venv-name/lib/python3.8/site-packages/numpy/core/fromnumeric.py", line 2324, in sum
    return _wrapreduction(a, np.add, 'sum', axis, dtype, out, keepdims=keepdims,
  File "/zhome/87/8/147075/venv-name/lib/python3.8/site-packages/numpy/core/fromnumeric.py", line 70, in _wrapreduction
    passkwargs = {k: v for k, v in kwargs.items()
  File "/zhome/87/8/147075/venv-name/lib/python3.8/site-packages/numpy/core/fromnumeric.py", line 71, in <dictcomp>
    if v is not np._NoValue}
KeyboardInterrupt
GPAW CLEANUP (node 9): <class 'KeyboardInterrupt'> occurred.  Calling MPI_Abort!
Error in sys.excepthook:
Traceback (most recent call last):
  File "/zhome/87/8/147075/venv-name/gpaw/gpaw/mpi.py", line 1197, in print_mpi_stack_trace
    def print_mpi_stack_trace(type, value, tb):
KeyboardInterrupt

Original exception was:
Traceback (most recent call last):
  File "soc.py", line 62, in <module>
    en_soc = soc_eigenstates(calc=calc, projected=True, theta=theta, phi=phi,
  File "/zhome/87/8/147075/venv-name/gpaw/gpaw/spinorbit.py", line 500, in soc_eigenstates
    dVL_avii = {a: soc(calc.wfs.setups[a],
  File "/zhome/87/8/147075/venv-name/gpaw/gpaw/spinorbit.py", line 500, in <dictcomp>
    dVL_avii = {a: soc(calc.wfs.setups[a],
  File "/zhome/87/8/147075/venv-name/gpaw/gpaw/spinorbit.py", line 545, in soc
    v_g = get_radial_potential(a, xc, D_sp)
  File "/zhome/87/8/147075/venv-name/gpaw/gpaw/spinorbit.py", line 610, in get_radial_potential
    fh_g = -np.array([np.sum(rho_g[:ig]) for ig in range(len(r_g))]) / r_g**2
  File "/zhome/87/8/147075/venv-name/gpaw/gpaw/spinorbit.py", line 610, in <listcomp>
    fh_g = -np.array([np.sum(rho_g[:ig]) for ig in range(len(r_g))]) / r_g**2
  File "<__array_function__ internals>", line 200, in sum
  File "/zhome/87/8/147075/venv-name/lib/python3.8/site-packages/numpy/core/fromnumeric.py", line 2324, in sum
    return _wrapreduction(a, np.add, 'sum', axis, dtype, out, keepdims=keepdims,
  File "/zhome/87/8/147075/venv-name/lib/python3.8/site-packages/numpy/core/fromnumeric.py", line 86, in _wrapreduction
    return ufunc.reduce(obj, axis, dtype, out, **passkwargs)
KeyboardInterrupt
GPAW CLEANUP (node 10): <class 'KeyboardInterrupt'> occurred.  Calling MPI_Abort!
